README.md

```diff
@@ -1,3 +1,23 @@
+## Merith-TK `sshcode` Fork
+___
+This fork is specifically for getting `sshcode` to work within the `msys`/`MinGW` enviroment
+
+What currently works, 
+* pretty much everything you would expect if you dont have chrome installed, starts the reverse proxy server, opens default browser.
+
+What doesnt work
+* Chrome support, cannot test as i dont use chrome.
+* Specifying a file path, for some odd reason, its still using windows filepath when trying to access the server with something like `sshcode --skipsync merith@192.168.0.101:/opt` will try and access `C:/mingw/opt` which, of course, does not exist. assistance is highly welcome.
```

Comment:
#118 handled this by translating the path. The original opener said they'd reopen it but it seems they've forgotten.

Reply:
@eargollo has agreed to merge our seperate code bases eventually,